release_notes.md

```diff
@@ -7,6 +7,7 @@
 ### Microsoft.Azure.Functions.Worker (metapackage) <version>
 
 - `AZURE_FUNCTIONS_` environment variables are now loaded correctly when using `FunctionsApplicationBuilder`. (#2878)
+- Return null when cosmos db input can't find a document (#2545)
```

Comment:
Release notes for the cosmos extension go here: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.CosmosDB/release_notes.md

```suggestion
- Return a successful result with a null value when  a Cosmos document cannot be found (#2943)
```